geode-core/src/main/java/org/apache/geode/distributed/internal/membership/NetView.java

```diff
@@ -640,4 +662,12 @@ private void writeAsArrayList(List list, DataOutput out) throws IOException {
   public int getDSFID() {
     return NETVIEW;
   }
+
+  public Map<String, Long> getMembersTimeout() {
+    return membersTimeout;
+  }
+
+  public void setMembersTimeout(Map<String, Long> membersTimeout) {
```

Comment:
This could cause the introduction of a non-concurrent map.  It doesn't look like this method is used so I think it should be deleted.